### PR TITLE
Feature/intellisense summaries

### PR DESCRIPTION
**Pull request by BellringerQuinn**

### CODE REVIEW

File: Assets/SequenceSDK/Authentication/ILogin.cs [hunk 1]
```diff
@@ -68,6 +68,10 @@ public interface ILogin
         /// </summary>
         public void AppleLogin();
 
+        /// <summary>
+        /// Returns true iff there is an authentication attempt in process
```
Comment (taylanpince) [inline Assets/SequenceSDK/Authentication/ILogin.cs:72]: iff > if

File: Assets/SequenceSDK/Authentication/IValidator.cs [hunk 2]
```diff
@@ -9,6 +9,11 @@ public interface IValidator
         /// <returns></returns>
         public bool ValidateEmail(string email);
         
+        /// <summary>
+        /// Returns true iff an OTP code is in correct form
```
Comment (taylanpince) [inline Assets/SequenceSDK/Authentication/IValidator.cs:13]: iff > if

File: Assets/SequenceSDK/Ethereum/Contract/ContractDeployer.cs [hunk 3]
```diff
@@ -24,6 +34,12 @@ public static class ContractDeployer
             return result;
         }
 
+        /// <summary>
+        /// Used to precalculate the contract deployment address
```
Comment (taylanpince) [inline Assets/SequenceSDK/Ethereum/Contract/ContractDeployer.cs:38]: "Used to" unnecessary, to standardize language

**Review by taylanpince — CHANGES_REQUESTED** [review]
Minor adjustments

File: Assets/SequenceSDK/Authentication/ILogin.cs (hunk 1 above)
Comment (BellringerQuinn) [reply Assets/SequenceSDK/Authentication/ILogin.cs:72]: I was thinking iff as in the short form for "if and only if". I guess that's not as common nomenclature as I thought it was. I'll change it thanks

Comment (taylanpince) [reply Assets/SequenceSDK/Authentication/ILogin.cs]: Ah I hadn't heard about it before, TIL!

**Review by taylanpince — APPROVED** [review]

**Pull request merged**